.github/workflows/testCoverage.yml

```diff
@@ -1,11 +1,71 @@
 name: 'Test coverage'
 on:
+  schedule:
+    - cron: "0 0 * * *" 
   workflow_dispatch:
-
+  
 jobs:
-  test_env_setup_job:
-    runs-on: ubuntu-latest
-    name: Test coverage
+  run_tests_windows:
+    name: Test coverage in windows-latest
+    strategy:
+      matrix:
+        os: [windows-latest]
+    runs-on: ${{ matrix.os }}
+    outputs:
+      test_coverage: ${{ steps.run_tests.outputs }}
+    steps:                                                   
+      - name: Checkout [master]
+        uses: actions/checkout@v4
+
+      - name: Checkout [react-charting]
+        uses: actions/checkout@v4
+        with: 
+          repository: "microsoft/fluentui"
+          path: repo1
+
+      - name: Show current directory
+        run: echo "$PWD" && ls
+
+      - name: Change function visibility from private to public
+        id: setup
+        uses: ./tools/UnitTestApp/ChangeFunctionVisibility
+        with:
+          osType: ${{ matrix.os }}
+
+      - name: Install packages
+        run: yarn --cwd ./tools/UnitTestApp && yarn --cwd ./repo1 && yarn --cwd ./repo1/packages/react-charting
```

Comment:
Can code duplication be avoided here

Reply:
Had to keep 3 jobs separate for parallelization. Don't think we can separate this out in functions.

Reply:
See how the steps are being used in template form in fluent ci pipeline https://github.com/microsoft/fluentui/blob/master/azure-pipelines-ci.yml

Reply:
Found that when we use a matrix strategy in GitHub workflow to run jobs across multiple operating systems, architectures, etc, GitHub Actions will run these jobs in parallel by default in https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs. So, instead of templatizing the code, have removed duplicate code by using the matrix strategy.